docling/datamodel/pipeline_options.py

```diff
@@ -214,7 +214,7 @@ class PictureDescriptionBaseOptions(BaseOptions):
     scale: float = 2
 
     bitmap_area_threshold: float = (
-        0.2  # percentage of the area for a bitmap to processed with the models
+        0.01  # percentage of the area for a bitmap to processed with the models
```

Comment:
We cannot allow to change this existing constant, it will change the default behaviour with our OCR.

Reply:
This constant appears to be unused. There is another `bitmap_area_threshold` under `ocr_options` which is separate and unchanged.

`0.2` is a very high threshold for picture description, many important images take up less than 1/5 of a page. I think this will be surprising to users.

Reply:
Indeed it isn't used so far! Since it isn't used we can actually take the opportunity to change the field name to `picture_area_threshold`, which is more accurate (there is not necessarily a bitmap involved). 
Also let's put 5% as a default then (0.05).

Reply:
done